src/docs/linea.mdx

```diff
@@ -110,7 +138,7 @@ Linea is a ZK-Rollup powered by Consensys zkEVM. The protocol team centrally ope
 
     | Opcode | Name | Solidity Equivalent | Rollup Behaviour | Ethereum L1 Behaviour |
     | :----- | :--- | :-------------------| :--------------- | :-------------------- |
-    | 44 | DIFFICULTY / PREVRANDAO | `block.difficulty` | Returns a fixed number | Get the output of the randomness beacon provided by the beacon chain <Modified /> |
+    | 44 | PREVRANDAO (legacy DIFFICULTY) | `block.difficulty` | Returns a fixed number | Get the output of the randomness beacon provided by the beacon chain <Modified /> |
     | 5F | PUSH0 |  | Not supported yet! <br /><br /> Solidity version `0.8.20` or higher can only be used with an evm version lower than the default `shanghai`. <br /><br /> Versions up to `0.8.19` (included) are fully compatible | Places value 0 on the stack <Unsupported /> |
 
 </Section>
```

Comment:
If the precompiles are working as in Ethereum L1 but they are not part of the validity proof lets add a "callout" similar to the zkSync warning in the OPCODES section:

"All Precompiled Contacts defined in the canonical Ethereum L1 implementation (until London hard fork) have the same behaviour on the Rollup, however, the execution of those Precompiles is not covered by the circuits and therefore not present in the Validity proof"

Reply:
Based on the answer that we receive from the TG group we can list the specific precompiles that are covered on mainnet not - if there are such